sspl_test/run_test.py

```diff
@@ -92,6 +92,7 @@ def tmain(argp, argv):
             init(args)
         except Exception as e:
             print('FAILED: Error: %s #@#@#@' %e)
+            print(traceback.format_exc())
```

Comment:
This will traceback for failed test only right, and not impacthe final test results list shown at last? In which case am ok

Reply:
Yes, only for the failed test. Final test result will come as usual.